### PR TITLE
Done with the Job first page, still have a bug to fix and on first pa…

### PR DESCRIPTION
**Pull request by Ilesanmi-Erioluwa-Victor**
…ge, will start Job detail page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +29,9 @@
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
     "@vitejs/plugin-react-swc": "^3.0.0",
+    "autoprefixer": "^10.4.14",
+    "postcss": "^8.4.23",
+    "tailwindcss": "^3.3.2",
```
Comment (copy-ninja1) [inline package.json:34]: please no tailwind use chakara for everything

File: src/components/Nav/index.tsx [hunk 2]
```diff
@@ -20,7 +20,7 @@ function Nav(): JSX.Element {
   return (
     <Box {...dflex} paddingBottom="1.5rem" width="100%">
```
Comment (copy-ninja1) [inline src/components/Nav/index.tsx:21]: delete this component

**Review by copy-ninja1 — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -12,7 +12,9 @@
     "@chakra-ui/react": "^2.5.5",
     "@emotion/react": "^11.10.6",
     "@emotion/styled": "^11.10.6",
+    "@fontsource/actor": "^4.5.9",
     "@fontsource/inter": "^4.5.15",
+    "@heroicons/react": "^2.0.17",
```
[the remark this answers is not in the document]
Comment (Ilesanmi-Erioluwa-Victor) [reply package.json:17]: Okay boss, i will do that

File: src/components/Nav/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (Ilesanmi-Erioluwa-Victor) [reply src/components/Nav/index.tsx:21]: I was not the one that worked on Nav component

**Comment by copy-ninja1** [comment]
delete the package-lock.json, allways use yarn

**Pull request merged**